project/dexter-vs/Dexter/Common/Core/Utils/DexterConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dexter.Common.Utils
+{
+    static public class DexterConstants
```

Comment:
Yes, but do we really need to set  `pfCancel`parameter in OnQueryCloseSolution() etc. ?
As far as I know there is no need to do it.